src/Bitmovin/api/model/encodings/StartEncodingManifestItem.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Bitmovin\api\model\encodings;
+
+use Bitmovin\api\model\ModelInterface;
+use JMS\Serializer\Annotation as JMS;
+
+class StartEncodingManifestItem implements ModelInterface
```

Comment:
Suggestions for name of this class?

Reply:
https://github.com/bitmovin/bitmovin-python/tree/master/bitmovin/resources/models/encodings/start/manifests That's how it is defined in the Python API client